draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -281,7 +281,7 @@ If a provider has an application-specific need to have multiple validations for
 
 This again allows the provider to query only for application-specific records it needs, while giving flexibility to the user adding the DNS record (i.e. they can be given permission to only add records under a specific prefix by the DNS administrator). Whether or not multiple validation records can exist for the same domain is up to the provider's application specification.
 
-Consumers of the provider services need to relay information from a provider's website or APIs to their local DNS administrators. The exact DNS record type, content and location is often not clear when the DNS administrator receives the information, especially to consumers who are not DNS experts. Providers SHOULD offer detailed help pages, that are accessible without needing a login on the provider website, as the DNS administrator often has no login account on the provider service website. Similarly, for clarity, the exact and full DNS record (including a Fully Qualified Domain Name) to be added SHOULD be provided along with help instructions.
+Validation records need to be relayed from a provider to a DNS administrator. Providers SHOULD offer detailed and easily-accessible help pages, keeping in mind that the DNS administrator might not have a login account on the provider's website. Similarly, for clarity, the exact and full DNS record (including a Fully Qualified Domain Name) to be added SHOULD be provided along with help instructions.
```

Comment:
```suggestion
Validation records need to be relayed from a provider to a DNS administrator. Providers SHOULD offer detailed and easily-accessible help pages, keeping in mind that the DNS administrator might not have a login account on the provider's website. Similarly, for clarity, the exact and full DNS record (including a Fully Qualified Domain Name and DNS record type) to be added SHOULD be provided along with help instructions.
```

We probably do want to be explicit about the DNS record type being included.
We seem to use "Provider" elsewhere so if we want to change we'd want to do so consistently?

Reply:
"service provider" exists in multiple places, though "provider service" exists once (and some icky grammar). 

If I have a vote I would say if we change Provider to 'Service Provider'.   It's doesn't allow mixing it up when "DNS Provider"  but that is me

Reply:
Let's do a separate editorial pass to change Provider to Service Provider if needed, doesn't need to be done in this PR. 
I'm not a 100% sure about the API line either, mainly because it's a bit vague.

Reply:
I'll open an issue on that and do some text crawls